common/lib/xmodule/xmodule/video_module/video_module.py

```diff
@@ -861,6 +862,15 @@ def student_view_data(self, context=None):
                     "file_size": 0,  # File size is unknown for fallback URLs
                 }
 
+        # Include youtube link if there is no encoding for mobile- ie only a fallback URL or no encodings at all
+        # We are including a fallback URL for older versions of the mobile app that don't handle Youtube urls
+        if self.youtube_id_1_0 and "mobile_low" not in encoded_videos:
+            youtube_1_0_url = "https://www.youtube.com/watch?v={}".format(self.youtube_id_1_0)
```

Comment:
I looked for helper methods but the one I found requires an edX video ID.

I look for `self.youtube_id_1_0` because I assumed that no course would make a sped-up or slowed-down version of  a video available without including the original video. If that's not a fair assumption I can look for all of the possible speeds, starting with 1x.